misc/helm-charts/operator/README.md.gotmpl

```diff
@@ -64,7 +64,7 @@ Setup process:
 
 #### Storage Configuration
 
-Once LVM is configured, set up the storage class:
+Once LVM is configured, set up the storage class in misc/helm-charts/operator/values.yaml:
```

Comment:
nit: It does not necessarily need to be in that file, people can create their own values files and override some of the values, here is an example:

https://github.com/MaterializeInc/terraform-aws-materialize/pull/3/files#diff-dcd4983a14d3d81cf5512e99948b63e7210e8fb84196fee7d83fb7713422c27aR72